Content.Server/GameObjects/EntitySystems/CrematoriumEntityStorageSystem.cs

```diff
@@ -0,0 +1,26 @@
+using Content.Server.GameObjects.Components.GUI;
+using Content.Server.GameObjects.Components.Morgue;
+using Content.Shared.GameObjects.EntitySystems;
+using Robust.Shared.Containers;
+using Robust.Shared.GameObjects;
+
+namespace Content.Server.GameObjects.EntitySystems
+{
+    internal sealed class CrematoriumEntityStorageSystem : EntitySystem
+    {
+        public override void Initialize()
+        {
+            base.Initialize();
+            SubscribeLocalEvent<HandsComponent, BlockDropHandItemsEvent>(HandleDropItems);
```

Comment:
This is my problem with the directed bus because now hands is going to need a bunch of component specific checks OR requires the use of broadcast events.